<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="demosaicMosaic">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Gråtonemosaikk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicOriginal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="demosaicOverskrift">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Demosaicing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Informasjonen i originalbildet &lt;br/&gt;legges over i en gråtonemosaikk,&lt;br/&gt;før det rekonstrueres ved hjelp&lt;br/&gt;av demosaicing.&lt;br/&gt;&lt;br/&gt;Velg til høyre hvilket bilde som&lt;br/&gt;skal brukes. Under bildet velges&lt;br/&gt;om man vil bruke egenskrevet&lt;br/&gt;kode, eller om man vil bruke&lt;br/&gt;kode med ferdige biblioteker.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicCode">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicDemosaic">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Demosaiced</string>
   </property>
  </widget>
  <widget class="QLabel" name="demosaicImg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>76</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../hdr-bilder/Adjuster/Adjuster_00032.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="adjuster">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjuster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="garden">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>200</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Garden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trees">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Trees</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stillife">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stillife</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mountains">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Mountains</string>
   </property>
  </widget>
  <widget class="QPushButton" name="balls">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Balls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fog">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fog</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ocean">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ocean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicDemosaicPackage">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>420</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Demosaiced</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicMosaicPackage">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Gråtonemosaikk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicOriginal2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Uten pakker:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bruk av pakker:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
